content/docs/reference/pkam.md

```diff
@@ -9,8 +9,38 @@ description: | # SEO Description of the page (Shows in google and atsign.dev sea
     Definition of PKAM (public key authentication mechanism)
 
 draft: false # Change this to "true" to hide the page
-toc: false # Change this to "true" to show the table of contents
+toc: true # Change this to "true" to show the table of contents
 weight: 208 # For single pages, lower is first.
 ---
 
-TODO PKAM goes here
+## Definition
+ PKAM stands for public key authentication mechanism. It is a mechanism for encrypting/decrypting data between two parties.
+
+## @platform
+
+The @platform's packages and SDKs help apps generate a RSA PKAM public/private keypair and save this in a `.atKeys` file containing these keys. Since Atsign provided the CRAM secret, this poses an invulnerability (because Atsign essentialy generated your password for you). To oppose this invulnerability, an RSA PKAM keypair is generated for subsequent authentications and the CRAM secret is voided. 
+
+The PKAM public key is given out to other clients who want to send data to you. The data they send to you is encrypted using your public key and is decrypted using your private key. Read more on asymmetric encryption {{< a target="_self" href="/docs/reference/encryption" >}} here. {{< /a >}}. 
+
+The PKAM private key used by the client to authenticate into their own secondary server. It is just like the CRAM secret except it is now owned by you. 
+
+## @protocol
+
+In this process, a {{< a target="_self" href="/docs/reference/pkam" >}} PKAM {{< /a >}} keypair was generated for subsequent logins to your secondary. PKAM keys will be generated by your device (which is an RSA public/private keypair). Then, a `.atKeys` file should be generated (and kept safely by users) for subsequent logins in your apps. Since the PKAM keypair will now be used to login to your apps, the CRAM secret is no longer valid for authentication into your secondary server.
+{{< a target="_blank" href="https://pub.dev/documentation/at_server_spec/latest/verbs/Cram-class.html" >}} Learn more on CRAM {{< /a >}}
+
+In the {{< a target="_self" href="/docs/atplatform/specification" >}} @protocol, {{< /a >}} PKAM is a verb.
```

Comment:
No need to say `target="_self"` since the browser will default to that if it is left undefined. You don't need to change, but just something to note.